api/src/features/system/system.py

```diff
@@ -0,0 +1,60 @@
+import json
+
+from starlette import status
+from fastapi import APIRouter, Request
+
+from restful.responses import create_response
+from features.system.use_cases.create_application_use_case import create_application_use_case
+from utils.logging import logger
+from starlette.responses import JSONResponse, PlainTextResponse
+from features.system.use_cases.get_application_settings_use_case import get_application_settings_use_case
+from features.system.use_cases.set_application_settings_use_case import set_application_settings_use_case
+
+router = APIRouter(tags=["System"], prefix="/system")
```

Comment:
```suggestion
router = APIRouter(tags=["Default", "System"], prefix="/system")
```
Consider adding "Default" (or some other shared tag) to each instance of `APIRouter` in order to generate a single API client when using `openapi-generator-cli` - see https://github.com/equinor/data-modelling-storage-service/pull/238#issuecomment-1205283810